### ISSUE TITLE
Preview Window Not Opening

### ISSUE THREAD

**Issue by borth79** [smeighan/xLights#2291]
**Describe the bug**
In version 2020.42 and 2020.43, the preview window does not open.

**To Reproduce**
1. Open xLights
2. Open Sequence
3. Open Preview (View -> Previews -> <custom preview>

**Expected behavior**
When click on View -> Previews -> <name of preview>, I would expect that the preview window open.

**Versions (please complete the following information):**
 - OS: Ubuntu 2020.04
 - xLights versions 2020.42 and 2020.43

**Additional context**
The original preview was generated in 2020.23. This preview window opens correctly in 2020.23 and 2020.40. It looks like there is now a checkbox in the Previews sub menu which indicates that an update was made.

**Comment by keithsw1111** [comment]
Odds are it is opening on another screen. Try dock all or reset positions. If it is still MIA then maybe it is a linux issue but these are the most common causes.

**Comment by borth79** [comment]
Sorry it took so long to reply. 
I misstated that this had worked in 2020.23. It had worked in 2019.23.

I believe I now know what the issue is. When opening a preview, the window was approximately 4px x 40px. It was just a little sliver on the screen. Screenshot attached. Once I found it, I was able to drag the borders and make it larger.

![Screenshot from 2020-10-28 19-02-08](https://user-images.githubusercontent.com/54193062/97512963-825ad100-1950-11eb-877a-a98ea2470112.png)
